exercises/pangram/example.h

```diff
@@ -0,0 +1,11 @@
+#if !defined(PANGRAM_H)
+#define PANGRAM_H
+#include <string>
+
+namespace pangram
+{
+    
+bool is_pangram(std::string s);
```

Comment:
Actually please make the string arguments for both of these functions take `std::string const& s`.

I missed this on my first pass...however this and the config.json update are the only things left :)

Reply:
latest push should have everything.